src/Gui/NaviCube.cpp

```diff
@@ -214,6 +215,8 @@ class NaviCubeImplementation : public ParameterGrp::ObserverType {
     bool m_MightDrag = false;
     double m_BorderWidth;
     NaviCube::Corner m_Corner = NaviCube::TopRightCorner;
+    bool m_RotateToNearest = true;
```

Comment:
Where is this member ever used? There are only methods to set the value but nowhere in the code the value is used.

Reply:
Now it is used.